package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ukis-frontend-libraries",
-  "version": "7.3.1",
+  "version": "7.3.2-next.0",
```

Comment:
Ah this could be the reason why the latest version for your tag was not published.

Have you only pushed the tag https://github.com/dlr-eoc/ukis-frontend-libraries/releases/tag/v7.3.2-next.0 and not created it with `npm version ...` like described here https://github.com/dlr-eoc/ukis-frontend-libraries/blob/master/DEVELOPMENT.md#alternative-prereleases-can-be-created?

Reply:
Hmm, interesting! I honestly cannot remember - but I'm not quite certain about that. 
When inspecting the [tag's source]( https://github.com/dlr-eoc/ukis-frontend-libraries/releases/tag/v7.3.2-next.0 ) I *do* find an updated version in package.json:
```
 https://github.com/dlr-eoc/ukis-frontend-libraries/releases/tag/v7.3.2-next.0 
```
But when looking at the [merged changes for that release branch](https://github.com/dlr-eoc/ukis-frontend-libraries/pull/76/files#diff-a0038a95dd42a57b4224f2c7d941dda7449e47e1c27395252e5ad168207b0cd2) I find *no* new version in the package.json

Reply:
Ok we should check this on the next release!